README.md

```diff
@@ -86,63 +70,51 @@ Prior to running this playbook some infrastructure needs to be in place
       /astores                      <- location for astores
   ```
 
-### Installation
+## Getting Started
+
+### Clone this project
 
-#### Ansible
+Run these commands in a Terminal session:
 
 ```bash
-# clone repo
-git clone https://github.com/sassoftware/viya4-deployment.git
+# clone this repo
+git clone https://github.com/sassoftware/viya4-deployment
 
 # move to directory
 cd viya4-deployment
-
-# install python packages
-pip3 install --user -r requirements.txt
-
-# install ansible collections
-ansible-galaxy collection install -r requirements.yaml -f
 ```
 
-#### Docker
+### Authenticating Ansible to access Cloud Provider
 
-```bash
-# clone repo
-git clone https://github.com/sassoftware/viya4-deployment.git
+See [Ansible Cloud Authentication](./docs/user/AnsibleCloudAuthentication.md) for details.
 
-# move to directory
-cd viya4-deployment
+**NOTE:** At this time, only required for GCP with external postgres
 
-# build container
-docker build -t viya4-deployment .
-```
-
-## Getting Started
-
-### Configs
+### Customize Input Values
 
 The playbook uses ansible vars for configuration. It is recommended to encrypt both this file and the other configs (sitedefault, kubeconfig, and tfstate) using ansible vault.
 
-### Ansible Vars
+#### Ansible Vars
 
 The ansible vars file is the main configuration file. Create a file named ansible-vars.yaml to customize any input variable value. For starters, you can copy one of the provided example variable definition files in ./examples folder. For more details on the variables declared in [ansible-vars.yaml](examples/ansible-vars.yaml) refer to [CONFIG-VARS.md](docs/CONFIG-VARS.md).
```

Comment:
"Example variable definition files are provided in the ./examples folder."

Reply:
updated